options/options.go

```diff
@@ -107,6 +113,9 @@ func New() (*Options, error) {
 	}
 
 	opts.SetPublishResults()
+
+	opts.ScorecardOpts.ResultsFile = envVars.EnvInputResultsFile
```

Comment:
Similar comment:
INPUT_ vars should be last resort.
Default to `SCORECARD_RESULTS_FORMAT` (`opts.ScoreOpts.Format`) and only prefer if it is unset or empty.

ref:
- https://github.com/ossf/scorecard/pull/1696
- https://github.com/ossf/scorecard/blob/c8acf3645f88b0dcbc5e798fe4b09730b869997b/options/options.go#L29-L52